src/commands/setup.rs

```diff
@@ -31,7 +31,12 @@ impl Setup {
 
         let network_options = match network::types::NetworkOptions::load(&input_file) {
             Ok(opts) => opts,
-            Err(e) => panic!("{}", e),
+            Err(e) => {
+                return Err(Box::new(network::error::NetavarkError {
+                    error: format!("Netavark Error {}", e),
```

Comment:
Please do not add `Netavark Error` to the error message. This provides zero context, when I call netavark I know I got an error from netavark. This message should be something like `failed to load network options: {}, err`